nuxeo-importer-core/src/main/java/org/nuxeo/ecm/platform/importer/base/GenericThreadedImportTask.java

```diff
@@ -233,6 +238,12 @@ protected DocumentModel doCreateLeafNode(DocumentModel parent, SourceNode node)
                 uploadedKO += fileSize;
             }
 
+            //
```

Comment:
Make the comment meaningful, like `// send an event about the imported document`